Gemfile

```diff
@@ -1,5 +1,5 @@
 source 'https://rubygems.org'
 
-gem 'cocoapods', '~> 1.10.0'
+gem 'cocoapods', '~> 1.11.3'
```

Comment:
We can probably just kill this now, honestly.

Reply:
We still want to keep this since this is a public library and some folks are still on CocoaPods!

Reply:
I....thought this was a PR on a different project when I made that comment. 😂🤦🏻‍♂️ Yep, you're right.